### ISSUE TITLE
Assert crash while running AsmJs/argTest.js

### ISSUE THREAD

**Issue by obastemur** [Microsoft/ChakraCore#4090]
Xplat CI CCRobot / Linux / ChakraCore Master

```
out/Debug/ch -WERExceptionSupport -ExtendedErrorStackForTestHost -BaselineMode -maxInterpretCount:1 -maxSimpleJitRunCount:1 -bgjit- -dynamicprofilecache:profile.dpl.137 -RecyclerVerifyMark -RecyclerConcurrentStress -VerifyBarrierBit -KeepRecyclerTrackData -maic:0 -maxInterpretCount:0 test/AsmJs/argTest.js

Output:
----------------------------
ASSERTION 19183: (lib/Runtime/Base/FunctionBody.cpp, line 1852) Cannot shrink register usage
 Failure: (m_inParamCount <= newInParamCount)

----------------------------
exit code: -4
```

**Comment by obastemur** [comment]
TODO: Enable disabled xplat test. -> https://github.com/Microsoft/ChakraCore/pull/4522

**Comment by MikeHolman** [comment]
@obastemur How often does this fail? I'm running that command locally and don't repro. I wonder if this has been fixed by #4531.

**Comment by obastemur** [comment]
@MikeHolman right, that PR has fixed this one too! Closing.
